tyk-pro/configs/tyk_analytics.conf

```diff
@@ -1,79 +1,43 @@
 {
-	"listen_port": 3000,
-	"tyk_api_config": {
-		"Host": "https://tyk-gateway.$namespace",
-		"Port": "443",
-		"Secret": "$sharedsecret"
-	},
-	"mongo_url": "$mongo",
-	"license_key": "$license",
-	"page_size": 10,
-	"admin_secret": "$adminsecret",
-	"shared_node_secret": "$sharedsecret",
-	"enable_cluster": false,
-	"force_api_defaults": false,
-	"notify_on_change": true,
-	"redis_database": 0,
-	"redis_hosts": {
-		"$redis": "$rport"
-	},
-	"hash_keys": true,
-	"email_backend": {
-		"enable_email_notifications": false,
-		"code": "",
-		"settings": null,
-		"default_from_email": "",
-		"default_from_name": ""
-	},
-	"hide_listen_path": false,
-	"sentry_code": "",
-	"sentry_js_code": "",
-	"use_sentry": false,
-	"enable_master_keys": false,
-	"enable_duplicate_slugs": true,
-	"show_org_id": true,
-	"host_config": {
-		"enable_host_names": true,
-		"disable_org_slug_prefix": true,
-		"hostname": "$dashboard.$slug.tyk.technology",
-		"override_hostname": "",
-		"portal_domains": {},
-		"portal_root_path": "/",
-		"generate_secure_paths": true
-	},
-	"http_server_options": {
-		"use_ssl": false,
-		"certificates": [{
-			"domain_name": "",
-			"cert_file": "",
-			"key_file": ""
-		}],
-		"min_version": 0
-	},
-	"ui": {
-		"login_page": {},
-		"nav": {
-			"dont_show_admin_sockets": true,
-			"hide_licenses_section": true
-		},
-		"uptime": {},
-		"portal_section": null,
-		"designer": {},
-		"dont_allow_license_management": true,
-		"dont_allow_license_management_view": true
-	},
-	"home_dir": "/opt/tyk-dashboard",
-	"identity_broker": {
-		"enabled": false,
-		"host": {
-			"connection_string": "",
-			"secret": ""
-		}
-	},
-	"tagging_options": {
-		"tag_all_apis_by_org": false
-	},
-	"use_sharded_analytics": false,
-	"enable_aggregate_lookups": true,
-	"aggregate_lookup_cutoff": "26/05/2016"
-}
\ No newline at end of file
+  "listen_port": 3000,
```

Comment:
This config file seems different. Does it still work the same way as before?

Reply:
Have tested the config and I believe it works like before.